docs/source/guides/contributing.md

```diff
@@ -56,8 +56,8 @@ how other languages implement the feature (if applicable), and so on.
 ## Submitting changes
 
 !!! note
-    Before submitting code changes, please note that we only accept merge
-    requests for issues labelled as "Accepting contributions".
+Before submitting code changes, please note that we only accept merge
+requests for issues labelled as "Accepting contributions".
```

Comment:
This too isn't necessary, and I think even incorrect given the syntax description at https://squidfunk.github.io/mkdocs-material/reference/admonitions/.